pkg/orchestrator/util.go

```diff
@@ -84,6 +85,11 @@ func (n NodeType) String() string {
 	}
 }
 
+func (n NodeType) BumpExpVar() {
+	e := forwarder.TransactionsIntakeOrchestrator[n]
+	e.Add(1)
+}
+
```

Comment:
It's a little bit weird to have a dependency on the forwarder in here, I suppose this will create a cyclic import issue since now the forwarder requires the NodeType struct and the NodeType struct requires the forwarder.

Better have separation of concerns and add an OrchestratorExpvarBundle or something like that for the map of expvars in the forwarder package directly that we could then use to bump stats for a given node type.

Reply:
I see your point. 
This is still WIP and should be seen about the idea